### PR TITLE
item identification: Add supplies in Tombs of Amascut

### PR DESCRIPTION
**Pull request by Molorius**
Item Identification plugin: Add supplies in Tombs of Amascut. Please let me know if any names should be changed.

### CODE REVIEW

**Comment by Molorius** [comment]
The item identification changes are mentioned in #15478. I forced a few commits to fix styling, change wording slightly, and to add Honey Locust.

**Review by Nightfirecat — CHANGES_REQUESTED** [review]
Why is there a new section added for new types of items? We don't do blanket item identification for all types of food/consumables, only potions. Please remove `SUPPLIES_TOA` and all non-potion items added here, and move the potion definitions to the appropriate section.

**Comment by Molorius** [comment]
I kept only the potions and moved them to the potions section. I just retested Nectar and Ambrosia, I can test the others if needed.

File: runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java [hunk 1]
```diff
@@ -325,6 +325,11 @@
 	MAGIC_ESS(Type.POTION, "MagEss", "M.E", ItemID.MAGIC_ESSENCE4, ItemID.MAGIC_ESSENCE3, ItemID.MAGIC_ESSENCE2, ItemID.MAGIC_ESSENCE1),
 	REJUVENATION(Type.POTION, "Rejuv", "Rj", ItemID.REJUVENATION_POTION_4, ItemID.REJUVENATION_POTION_3, ItemID.REJUVENATION_POTION_2, ItemID.REJUVENATION_POTION_1),
 
+	NECTAR(Type.POTION, "Nectar", "N", ItemID.NECTAR_4, ItemID.NECTAR_3, ItemID.NECTAR_2, ItemID.NECTAR_1),
+	AMBROSIA(Type.POTION, "Ambros", "A", ItemID.AMBROSIA_2, ItemID.AMBROSIA_1),
+	LIQUID_ADRENALINE(Type.POTION, "L.Adrn", "L.A", ItemID.LIQUID_ADRENALINE_2, ItemID.LIQUID_ADRENALINE_1),
+	TEARS_OF_ELIDINIS(Type.POTION, "TearsOE", "ToE", ItemID.TEARS_OF_ELIDINIS_4, ItemID.TEARS_OF_ELIDINIS_3, ItemID.TEARS_OF_ELIDINIS_2, ItemID.TEARS_OF_ELIDINIS_1),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java:331]:
```suggestion
	TEARS_OF_ELIDINIS(Type.POTION, "Tears", "ToE", ItemID.TEARS_OF_ELIDINIS_4, ItemID.TEARS_OF_ELIDINIS_3, ItemID.TEARS_OF_ELIDINIS_2, ItemID.TEARS_OF_ELIDINIS_1),
```

**Pull request merged**